src/common/redis_module/ray_redis_module.cc

```diff
@@ -394,17 +447,18 @@ bool PublishObjectNotification(RedisModuleCtx *ctx,
 int TableAdd_RedisCommand(RedisModuleCtx *ctx,
```

Comment:
This command seems very bloated now.  A rule of thumb I learned was any func > 50 lines should probably be broken down.

Reply:
Sounds good, although a lot of the bloat will hopefully go away soon, since a lot of the code is here is only necessary to support legacy code (like the task table).